scripts/generateGitHubStars.js

```diff
@@ -33,6 +37,8 @@ async function readHomepage(manifestFile) {
 }
 
 async function main(vcpkgDir, destDir, githubToken) {
+    try {
+    console.log("github token ", githubToken);
```

Comment:
We should not print the github token to standard out.

Reply:
for dev purpose only - now removing this log